daemon/algod/api/server/v2/dryrun_test.go

```diff
@@ -1091,3 +1091,102 @@ int 1`)
 		logResponse(t, &response)
 	}
 }
+
+func TestDryrunLogs(t *testing.T) {
+	t.Parallel()
+
+	ops, err := logic.AssembleString(`
+#pragma version 5
+int 1
+loop: byte "a"
```

Comment:
the idea is to use ascii char codes to obtain new char value on each loop iteration.

Reply:
Actually `itob` always produces an 8 byte slice. It would be better to so something like:
```
byte "a"
int 0
dup2
getbyte // convert 0th byte of "a" to integer, 97
int 1
+ // add 1
setbyte // set 0th byte of "a" to the byte given by the integer on the stack (97 + 1, "b")
```